dev/protos/firestore/bundle.proto

```diff
@@ -85,6 +88,9 @@ message BundleMetadata {
 
   // Time at which the documents snapshot is taken for this bundle.
   google.protobuf.Timestamp create_time = 2;
+
+  // The version of bundle.
```

Comment:
Nit: "The _schema_ version of _the_ bundle."

Reply:
Done.